### PR TITLE
feat: community section

### PR DESCRIPTION
**Pull request by Agustina-Carrion**

### CODE REVIEW

**Review by pReya — CHANGES_REQUESTED** [review]
@Agustina-Carrion This is still missing some of the fixes for stuff I mentioned on Slack already (stuff that I broke when merging all the branches).
Screenshot 2023-01-21 at 10 12 47

Screenshot 2023-01-21 at 10 12 43

**Comment by Agustina-Carrion** [comment]
I'm not sure how to fix it, because i don't have that broken section of the code. Images are showing for me 😅, and i tried different browsers. 

Screenshot 2023-01-21 at 11 40 36

File: src/components/CommunityBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  return (
+    <div
+      className={`container-width lg:container-width-hero h-auto mt-28 mb-10 lg:m-10 bg-cover bg-center ${
+        bgMapping[bg - 1]
+      } ${className || ""}`}
+    >
+      <div className="flex flex-col gap-3 md:w-[60%] h-full pl-4 pr-16 pb-10 pt-44 md:p-12 text-white items-start justify-end">
+        {title && (
+          <h1 className="font-semibold text-3xl font-aktiv">{title}</h1>
```
Comment (pReya) [inline src/components/CommunityBanner.tsx:25]: Font size is not matching Figma here. Figma calls for 32px and 40px line-height. This is not part of the Tailwind scale – so we should use the arbitrary value syntax here.

File: src/components/CommunityGridCard.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  title,
+  text,
+  target,
+}: Props) => {
+  return (
+    <div className={`col-span-1 bg-neutral-50 ${className}`}>
+      <div className="flex w-full h-full space-x-6 p-6">
+        <div className="flex-1 flex flex-col justify-between">
+          <div className="flex flex-col items-start gap-4">
+            <h2 className="text-2xl font-semibold text-neutral-900">{title}</h2>
```
Comment (pReya) [inline src/components/CommunityGridCard.tsx:21]: This is supposed to be `atkiv` font.

File: src/components/WideCard.tsx [hunk 3]
```diff
@@ -1,22 +1,56 @@
...
-      <div className="bg-tolo-green relative overflow-hidden">
-        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[60%] h-full flex flex-col justify-between">
-          {title && <h1 className="font-semibold text-5xl font-aktiv">{title}</h1>}
-          {children && <p className="font-medium text-2xl mt-8">{children}</p>}
-          <a href="#" className="bg-black z-10 inline-flex items-center bg-opacity-20 opacity-90 rounded-full py-4 px-5 mt-8">Search for Resources<ArrowIcon className="h-5 w-5 ml-8 text-white"/></a>
+    <div className={"container-width lg:container-width-hero mt-20 h-full"}>
+      <div className="bg-tolo-green relative overflow-hidden z-10">
+        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[65%] h-full flex flex-col justify-start gap-6 z-20">
+          {title && (
+            <h1 className="font-semibold text-4xl lg:text-5xl font-aktiv">
```
Comment (pReya) [inline src/components/WideCard.tsx:35]: This font size is supposed to be 48px according to Figma. It's not in the tailwind scale, so you need to use arbitrary value (or create a custom class in tailwind config – if this is used in multiple places).

File: src/components/WideCard.tsx [hunk 4]
```diff
@@ -1,22 +1,56 @@
...
   return (
-    <div className={`container-width-hero mt-20 h-full ${className || ""}`}>
-      <div className="bg-tolo-green relative overflow-hidden">
-        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[60%] h-full flex flex-col justify-between">
-          {title && <h1 className="font-semibold text-5xl font-aktiv">{title}</h1>}
-          {children && <p className="font-medium text-2xl mt-8">{children}</p>}
-          <a href="#" className="bg-black z-10 inline-flex items-center bg-opacity-20 opacity-90 rounded-full py-4 px-5 mt-8">Search for Resources<ArrowIcon className="h-5 w-5 ml-8 text-white"/></a>
+    <div className={"container-width lg:container-width-hero mt-20 h-full"}>
+      <div className="bg-tolo-green relative overflow-hidden z-10">
+        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[65%] h-full flex flex-col justify-start gap-6 z-20">
```
Comment (pReya) [inline src/components/WideCard.tsx:33]: Padding in large/desktop viewport is supposed to be 40px, not 48px.

File: src/components/WideCard.tsx [hunk 5]
```diff
@@ -1,22 +1,56 @@
...
+          {title && (
+            <h1 className="font-semibold text-4xl lg:text-5xl font-aktiv">
+              {title}
+            </h1>
+          )}
+          {text && <p className="font-medium text-2xl">{text}</p>}
+          {target && (
+            <ButtonLink
```
Comment (pReya) [inline src/components/WideCard.tsx:41]: There's a little too much space below the GitHub button in the desktop view.

**Review by pReya — CHANGES_REQUESTED** [review]

**Review by pReya — CHANGES_REQUESTED** [review]
Still missing a little bit more spacing here on the home page:
Screenshot 2023-01-21 at 12 23 58

File: src/components/CommunityBanner.tsx (hunk 1 above)
Comment (Agustina-Carrion) [reply src/components/CommunityBanner.tsx:25]: also added leading-10 for line-height!

File: src/components/CommunityBanner.tsx [hunk 6]
```diff
@@ -22,7 +22,7 @@ const CommunityBanner: React.FC<Props> = ({ className, text, title, bg }) => {
     >
       <div className="flex flex-col gap-3 md:w-[60%] h-full pl-4 pr-16 pb-10 pt-44 md:p-12 text-white items-start justify-end">
         {title && (
-          <h1 className="font-semibold text-3xl font-aktiv">{title}</h1>
+          <h1 className="font-semibold text-[32x] leading-10 font-aktiv">{title}</h1>
```
Comment (pReya) [inline src/components/CommunityBanner.tsx:25]: You're missing a `p` in `px`

**Review by pReya — CHANGES_REQUESTED** [review]
Unfortunately now, when we make the viewport smaller, the Button inside the WideCard can move out of the green box so it's not visible any more. I see two options to fix this:
Screenshot 2023-01-21 at 17 44 15

- Make paddings, font sizes smaller and content width wider on smaller viewports, so the text uses less vertical space and the Button doesn't move out of the green box
- Make the height of the green box dynamic/auto, so the button will always fit.

I guess you could also combine both solutions.

**Review by pReya — CHANGES_REQUESTED** [review]
This is probably unrelated to your changes, and might've been there before earlier, but I just noticed it: The two green boxes at the bottom cover up the nav bar. This is another `z-index` problem. Can you make sure the nav bar has a higher z-index than the green boxes?

Screenshot 2023-01-21 at 17 36 58

File: src/components/CommunityBanner.tsx (hunk 6 above)
Comment (Agustina-Carrion) [reply src/components/CommunityBanner.tsx:25]: 🙏 thank you for spotting that typo! it's fixed! 😁

**Review by pReya — CHANGES_REQUESTED** [review]

**Review by pReya — CHANGES_REQUESTED** [review]
Screenshot 2023-01-23 at 11 35 57

The header is too wide, again.

**Review by pReya — CHANGES_REQUESTED** [review]
Screenshot 2023-01-23 at 11 40 02

x-padding should be the same as y-padding and both should be 20px

**Review by pReya — CHANGES_REQUESTED** [review]
Screenshot 2023-01-23 at 12 00 34

Community and Tolocar Logo should be aligned on the left. This is not a problem with your code, but rather with my existing code. The SVG content in the logo is not left aligned.

**Review by pReya — CHANGES_REQUESTED** [review]
Screenshot 2023-01-23 at 12 02 45

Padding in the header on mobile viewport is too much.

**Review by pReya — CHANGES_REQUESTED** [review]
The background image of the hero section has now moved to the right of the viewport. It should be aligned to the right of the content-width instead.

**Pull request merged**